Client/Frontend/Browser/Tab.swift

```diff
@@ -256,6 +256,23 @@ class Tab: NSObject {
         contentScriptManager.helpers.removeAll()
     }
     
+    func clearHistory(config: WKWebViewConfiguration) {
+        guard let webView = webView, let firstWebsite = webView.backForwardList.backList.first else { return }
+
+        // First we make sure the webview is on the earliest item in the history
+        if webView.canGoBack {
+            webView.go(to: firstWebsite)
+        }
```

Comment:
Why would we force the user back to the first navigation item when they clear history?

Reply:
Actually this is not needed, I was using to clear the backwardlist first, will remove

Reply:
Unnecessary back navigation is removed